FROM python
USER root

RUN apt-get update

